Amazon Scraper/scrape.py

```diff
@@ -0,0 +1,33 @@
+import csv
+import pandas as pd
+from bs4 import BeautifulSoup
+import requests
+import time
+def scraper():
+    f=open('amazon.csv','w',newline='')
+    data=csv.writer(f)
+    headers = (
+        {
+        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36',
+        'Accept-language': 'en-US,en;q=0.5'
+        }      )
+    main=[]
+    preq=requests.get(url_name,headers)
+    soup=BeautifulSoup(preq.content,features="html.parser")
+    product_author_name=[i.text for i in soup.findAll("span",{"class":"a-size-small a-color-base"})]
+    data.writerow(product_author_name)
+    main.append(product_author_name)
+    product_rating=[ratings.text for ratings in soup.findAll("a",{"class":"a-size-small a-link-normal"})]
+    data.writerow(product_rating)
+    main.append(product_rating)
+    df=pd.DataFrame(data=main)
+    df.to_excel('amazon_report.xlsx',index=False,header=False)
```

Comment:
you are saving this as .xlsx but in the function start, you loaded a CSV? What is the ambiguity here?

Reply:
Sir  in this program you can use anything thing between to_csv or to_excel...The result will come out same...